app.rb

```diff
@@ -0,0 +1,87 @@
+require 'pg'
+require 'sinatra'
+require 'sinatra/reloader'
+require 'active_record'
+
+# Load the file to connect to the DB
+require_relative 'db/connection'
+
+# Load models
+require_relative 'models/apartment'
+require_relative 'models/tenant'
+
+get '/' do
+  erb :home
+end
+
+get '/apartments' do
+  @apartments = Apartment.all.order(address: :asc)
+  erb :"apartments/index"
+end
+
+get '/apartments/new' do
+  erb :"apartments/new"
+end
+
+post '/apartments' do
+  @apartment = Apartment.create(address: params[:address], monthly_rent: params[:monthly_rent], sqft: params[:sqft], num_beds: params[:num_beds], num_baths: params[:num_baths])
```

Comment:
you can also put `params[:apartment]` here instead of all that, but hardcore bonus points for writing all that out